app/scripts/components/alerts.component.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
```

Comment:
I'm a fan of moving things to a `scripts/components` directory, but would like to see a step further.  Can we do something like:

```
- scripts 
   - components 
     - alerts
        - alert.component.js
        - alert.view.html
```
I'm 150% into a component based heirarchy, its so much easier to derive relationships.  I realize this will require a little bit of tweaking to some of our build tools, but I doubt its much more than tweaking the globbing patterns.

@spadgett @jwforres may not agree, so I'll defer to their preference.

Reply:
I agree, I would probably go even one step further and get rid of the scripts folder:
```
app/
  components/
    alerts/
      alerts.component.js
      alerts.component.spec.js
      alerts.template.html
```
Also, should we think about breaking this out into a separate story? I feel like introducing file structure changes in increments that are too small (one component at a time) will be confusing.

Reply:
Agree on pulling it out of `scripts` since components are not 100% scripts.

We def prefer small PRs over big sweeping PRs, so I don't think this is a problem.  Rebases of giant structural PRs are terribly painful.

Reply:
Yeah, I was thinking it could still be incremental, just more than one file at a time. We could choose whole sections to do at a time and break it up into a few PRs.

Reply:
It seems like a good time to do this.